concent_api/core/message_handlers.py

```diff
@@ -302,7 +308,16 @@ def handle_send_force_subtask_results(request, client_message: message.concents.
             reason = message.concents.ServiceRefused.REASON.DuplicateRequest,
         )
 
-    if not base.is_provider_account_status_positive(request):
+    concent_rpc = new_sci_rpc(
```

Comment:
I think this should be instantiated:
1. only when payment backend is set to `sci_backend`.
2. only once.

That's why my proposal is to make it a singleton, instantiate it somewhere in `sci_backend` (and only when it is set), remove it from backend function calls (as it is specific only for `sci_backend`).

I know this sound like a decent change and some extra work, so I would like to ask @cameel and @dybi for opinion before starting this.

Reply:
@rwrzesien, I agree, we should have a function/class that would ensure only one backend exists. I think it could be in separate module, maybe `core/payments/helpers.py`?

Reply:
@dybi It works a little bit differently. There is a setting `PAYMENT_BACKEND` and you can set whichever you want on each environment. So there is only one used at a time, the issue here is that Karol is using code specific to `sci_backend` in view, so the question was should it be moved to `sci_backend`.

Reply:
@kbeker As discussed, we want to:
1. Instantiate it only when `sci_backend` is set.
2. Make it a singleton class, most probably as part of `sci_backend` module.

We can discuss implementation details if you want.

Reply:
Done :)